gateway-framework/src/http/middleware/require_auth.rs

```diff
@@ -139,13 +141,13 @@ where
             };
 
         match &auth_token {
-            AuthToken::StudioApiKey(auth) => tracing::info!(
-                target: reports::CLIENT_QUERY_TARGET,
+            AuthToken::ApiKey(auth) => tracing::info!(
+                target: "client_request",
```

Comment:
Why are we moving away from a constant here? I feel like there should be an explicit connection between the name selected here and the targets filtered for reports.

Reply:
Ah, yeah. If that should be a hard connection, then I'll change the code a little.

Reply:
The problem is that `tracing::info!` expect a constant literal for `target`. I haven't been able to pass down the `target` in any way, not even via an associated constant on an input struct.

@Theodus Would it be terrible to use `client_request` for _all_ incoming requests, regardless of the gateway and data service? Even hard-coding it in the target filtering for reports? Same with `indexer_request`.

Reply:
I think that's fine. As long as by "all incoming requests" you mean the primary requests associated with the data service. For example, the gateway currently serves other requests like requests for vouchers. But since those aren't handled by this code base for TAP, I think it's fine to ignore those for now when it comes to the name `client_request`.

Reply:
Yeah, those other requests are not or should not be going through `RequireAuthorizationLayer` and therefor wouldn't be reported as `"client_request"`s.